src/actions/notificationsActions.js

```diff
@@ -103,14 +101,6 @@ const onFirebaseMessageAction = (message: FirebaseMessage) => {
     if (type === FCM_DATA_TYPE.COLLECTIBLE) {
       dispatch(fetchAllCollectiblesDataAction());
     }
-
-    if (message.notification) {
-      const { wallet: { data: wallet } } = getState();
-      const notification = message.data && getToastNotification(message.data, wallet?.address);
```

Comment:
What if we replace this piece to get current account address and pass it to `getToastNotification` instead of key based wallet? This way we can have same functionality for Smart wallets when it will start working and the `getToastNotification`can remain, wdyt?

Reply:
@poocart Done with the changes.